bluesky/plan_tools.py

```diff
@@ -214,14 +253,12 @@ def ct(dets, n, subs=None):
     dets = list(dets)
     if subs is None:
         subs = {'all': [LiveTable(dets)]}
-    if n is None:
+    if n != 1:
         subs['all'].append(LivePlot(scalar_heuristic(dets[0])))
 
     plan = stage_wrapper(repeater(n, trigger_and_read, dets),
                          dets)
-    plan_with_subs = subscription_wrapper(plan, subs)
-
-    ret = yield from plan_with_subs
```

Comment:
@tacaswell I am wrong in thinking that these subscriptions are being add too late to see the RunStart document?